### PR TITLE
Changes to support simulation using weights for inference

### PR DESCRIPTION
**Pull request by souravsanyal06**
Made changes to simulator so that one can run simulation using real weights obtained from any framework and observe data sparsity in MVMUs. 

Follow the how_to_run.md for updated instructions

### CODE REVIEW

File: include/constants.py [hunk 1]
```diff
@@ -157,7 +157,7 @@
                     '8' : 1.8,
                     '16': 3.6}
 
-adc_pow_leak_dict = {'1' : 0.025,
+adc_pow_leak_dict = {'1' : 0.25,
```
Comment (Aayush-Ankit) [inline include/constants.py]: Typo.

File: src/ima_modules.py [hunk 2]
```diff
@@ -96,7 +102,10 @@ def propagate_dummy (self, inp = 'nil'):
             temp_inp = (cfg.num_bits - cfg.dac_res) * '0' + inp[i]
             inp_float[i] = fixed2float(temp_inp, cfg.int_bits, cfg.frac_bits)
         inp_float = np.asarray (inp_float)
+        #print('inp_float')
+        #print(inp_float)
```
Comment (Aayush-Ankit) [inline src/ima_modules.py]: Debug comments - remove.

Comment (souravsanyal06) [reply src/ima_modules.py]: removed

File: include/constants.py (hunk 1 above)
Comment (souravsanyal06) [reply include/constants.py]: fixed

**Pull request closed without merging**